### PR TITLE
homepage contents

### PR DESCRIPTION
**Pull request by shawn320**
# Description

Re-created the components and added them to the `Page` collection.

### CODE REVIEW

File: src/components/page-element/paragraph.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "collectionName": "components_page_element_paragraphs",
+  "info": {
+    "displayName": "Paragraph",
+    "icon": "align-justify"
+  },
+  "options": {},
+  "attributes": {
+    "Text": {
+      "type": "string"
```
Comment (zeddotes) [inline src/components/page-element/paragraph.json:10]: Is this a rich text or plain text input?

File: src/components/page-element/banner.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      "type": "string"
+    },
+    "Description": {
+      "type": "text"
+    },
+    "BannerImage": {
+      "allowedTypes": [
+        "images",
+        "files",
+        "videos"
```
Comment (zeddotes) [inline src/components/page-element/banner.json:20]: Should probably be images only. Maximum of 1 image if that can be set.

File: src/components/page-element/paragraph.json (hunk 1 above)
Comment (shawn320) [reply src/components/page-element/paragraph.json:10]: > Is this a rich text or plain text input?

Nice catch, this should be a rich text

**Review by zeddotes — APPROVED** [review]
Thanks for making the changes. Looks great.

**Pull request merged**